### PR TITLE
Update suivi.html

### PR DESCRIPTION
**Pull request by agarrone**
Update politique de confidentialité

### CODE REVIEW

File: udata_front/theme/gouvfr/templates/suivi.html [hunk 1]
```diff
@@ -17,42 +17,165 @@
...
+                <li>Le <a href="https://www.legifrance.gouv.fr/loda/id/JORFTEXT000039281619">décret n° 2019-1088 du 25 octobre 2019</a> relatif au système d’information et de communication de l’Etat et à la direction interministérielle du numérique, notamment son article 6 ;</li>
+                <li>La <a href="https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000024072788">circulaire du 26 mai 2011</a> relative à la création du portail unique des informations publiques de l’État “data.gouv.fr” ;</li>
+                <li>La <a href="https://www.legifrance.gouv.fr/circulaire/id/45162">circulaire du 27 avril 2021</a> relative à la politique publique de la donnée, des algorithmes et des codes sources.</li>
+            </ul>
 
-            <iframe class="well well-grey-100 fr-my-2w w-100" height="150"
-                src="https://stats.data.gouv.fr/index.php?module=CoreAdminHome&action=optOut&language=fr"></iframe>
+            <h2>Pendant combien de temps conservons-nous ces données ?</h2>
+            
+            <table>
```
Comment (nicolaskempf57) [inline udata_front/theme/gouvfr/templates/suivi.html:65]: Can you wrap the tables in a `<div class='fr-table'>`  to have the DSFR styling applied ?

File: udata_front/theme/gouvfr/templates/suivi.html [hunk 2]
```diff
@@ -17,42 +17,165 @@
...
+                Ce consentement de la personne concernée pour une ou plusieurs finalités spécifiques constitue une base légale au sens du RGPD et doit être entendu au sens de l'article 6-a du Règlement (UE) 2016/679 du Parlement européen et du Conseil du 27 avril 2016 relatif à la protection des personnes physiques à l'égard du traitement des données à caractère personnel et à la libre circulation de ces données.
+                Nous utilisons l’outil de mesure d’audience Matomo, configuré en mode exempté et ne nécessitant pas le recueil de votre consentement conformément aux recommandations de la CNIL.
+            </p>
+
+            <iframe class="well well-grey-100 fr-my-2w w-100" height="150"
+                src="https://stats.data.gouv.fr/index.php?module=CoreAdminHome&action=optOut&language=fr"></iframe>
+
+            <h2>Pour aller plus loin, vous pouvez consulter les fiches proposées par la CNIL :</h2>
+            <ul>
+                <li><a href="https://www.cnil.fr/fr/cookies-traceurs-que-dit-la-loi">*Cookies & traceurs : que dit la loi ?*</a></li>
```
Comment (nicolaskempf57) [inline udata_front/theme/gouvfr/templates/suivi.html:176]:
```suggestion
                <li><a href="https://www.cnil.fr/fr/cookies-traceurs-que-dit-la-loi">Cookies & traceurs : que dit la loi ?</a></li>
```

File: udata_front/theme/gouvfr/templates/suivi.html [hunk 3]
```diff
@@ -17,42 +17,165 @@
...
+                <li><a href="https://www.cnil.fr/fr/cookies-les-outils-pour-les-maitriser">*Cookies : les outils pour les maîtriser*</a></li>
```
Comment (nicolaskempf57) [inline udata_front/theme/gouvfr/templates/suivi.html:177]:
```suggestion
                <li><a href="https://www.cnil.fr/fr/cookies-les-outils-pour-les-maitriser">Cookies : les outils pour les maîtriser</a></li>
```

File: udata_front/theme/gouvfr/templates/suivi.html [hunk 4]
```diff
@@ -17,42 +17,165 @@
 <section class="default">
     <div class="fr-container fr-py-9v">
         <article class="suivi fr-col-12">
```
Comment (nicolaskempf57) [inline udata_front/theme/gouvfr/templates/suivi.html:19]:
```suggestion
        <article class="markdown fr-col-12">
```

File: udata_front/theme/gouvfr/templates/suivi.html [hunk 5]
```diff
@@ -17,42 +17,165 @@
...
+            </p>
 
-            <h2>Cookies déposés et opt-out</h2>
+            <h2>Quelles sont les données traitées ?</h2>
+            
+            <p class="text-justify">
+                Ne sont traitées que les données strictement nécessaires au fonctionnement du service. 
+                Ces données ne pourront en aucun cas servir au contrôle et à la surveillance de l’activité des Utilisateurs et Utilisatrices. Les données à caractère personnel concernées sont les suivantes :
+            </p>
+            <ul>
```
Comment (nicolaskempf57) [inline udata_front/theme/gouvfr/templates/suivi.html:32]:
```suggestion
            <ul class="fr-mb-3w">
```

File: udata_front/theme/gouvfr/templates/suivi.html [hunk 6]
```diff
@@ -17,42 +17,165 @@
...
+            <h2>Qu’est-ce qui nous autorise à traiter des données à caractère personnel ?</h2>
+            
             <p class="text-justify">
-                Ce site dépose un petit fichier texte (un « cookie ») sur votre ordinateur lorsque vous le
-                consultez.
-                Cela nous permet de mesurer le nombre de visites et de comprendre quelles sont les pages les plus
-                consultées.
+                <a href="http://data.gouv.fr">data.gouv.fr</a> traite des données à caractère personnel en se basant sur : 
             </p>
+            <ul>
```
Comment (nicolaskempf57) [inline udata_front/theme/gouvfr/templates/suivi.html:51]:
```suggestion
            <ul class="fr-mb-3w">
```

File: udata_front/theme/gouvfr/templates/suivi.html [hunk 7]
```diff
@@ -17,42 +17,165 @@
...
+                <li>L’exécution d’une mission d’intérêt public dont est investi le responsable de traitement au sens de l’article 6-1 e) du RGPD.</li>
+            </ul>
+            <p class="text-justify">
+                Cette mission d’intérêt public se traduit en pratique par : 
+            </p>
+            <ul>
```
Comment (nicolaskempf57) [inline udata_front/theme/gouvfr/templates/suivi.html:57]:
```suggestion
            <ul class="fr-mb-3w">
```

File: udata_front/theme/gouvfr/templates/suivi.html [hunk 8]
```diff
@@ -17,42 +17,165 @@
...
+                </tbody>
+            </table>
 
-            <h2>Ce site n'affiche pas de bannière de consentement aux cookies, pourquoi ?</h2>
+            <h2>Qui peut avoir accès à ces données ?</h2>
+            
+            <p class="text-justify">
+                Les accès aux données à caractère personnel sont strictement encadrés et juridiquement justifiés. Les personnes suivantes vont avoir accès aux données :
+            </p>
+            <ul>
```
Comment (nicolaskempf57) [inline udata_front/theme/gouvfr/templates/suivi.html:89]:
```suggestion
            <ul class="fr-mb-3w">
```

File: udata_front/theme/gouvfr/templates/suivi.html [hunk 9]
```diff
@@ -17,42 +17,165 @@
...
-                de la <abbr title="Commission Nationale de l'Informatique et des Libertés">CNIL</abbr>. Cela
-                signifie que votre adresse IP, par exemple, est anonymisée avant d'être enregistrée.
-                Il est donc impossible d'associer vos visites sur ce site à votre personne.
+                Vous disposez d’un droit d’accès et de rectification des données à caractère personnel qui vous concernent. Vous disposez également d’un droit d’opposition et de limitation du traitement de vos données.
+                Pour exercer vos droits ou pour toute question sur le traitement de vos données, vous pouvez nous écrire à l’adresse : <a href="http://support.data.gouv.fr">support.data.gouv.fr</a>.
             </p>
+            <p class="text-justify">
+                Vous pouvez également contacter la déléguée à la protection des données (DPD) des services du Premier Ministre : 
+            </p>
+            <ul>
```
Comment (nicolaskempf57) [inline udata_front/theme/gouvfr/templates/suivi.html:146]:
```suggestion
            <ul class="fr-mb-3w">
```

**Review by maudetes — APPROVED** [review]
Thank you both! I think you can add a changelog entry :blush:

**Pull request merged**